### PR TITLE
ci: add a workflow for weekly semantic versioning and release to google play

### PR DESCRIPTION
**Pull request by matthiasemde**

### CODE REVIEW

File: .github/workflows/testing-release.yml [hunk 1]
```diff
@@ -1,12 +1,232 @@
...
+      - name: Remove temporary tag
+        if: always()
+        run: |
+          git tag -d $tag_name
+          git push origin :refs/tags/$tag_name
+
+      - name: Commit changes and add a release tag
+        run: |
+          git commit -am "chore: bump version to ${{ env.version_name }}"
+          MESSAGES=("Beep Boop! New release coming through! 🤖" "I'm a bot and I approved this release. 🚀" "Another day, another tag. Life as a bot is exciting! 🎉" "Automated Release: I'm just a bot, but I'm pretty sure this is a good one! 🤖" "Release ready! All systems are go. 🚀" "New release deployed. I hope humans like it! 🎉" "Tag, you're it! New release out. 🏷️" "Just another day in the life of a release bot. Enjoy the new version! 📦")
```
Comment (maybeanerd) [inline .github/workflows/testing-release.yml:103]: This is great 👌🏻😂

File: build.properties [hunk 2]
```diff
@@ -1,3 +1,3 @@
 versionName=0.9.2
 versionCode=3
-commitHash=bastiistderbeste
\ No newline at end of file
+commitHash=debug
```
Comment (maybeanerd) [inline build.properties:3]: An era ends 😢

**Review by maybeanerd — COMMENTED** [review]
This is looking gooood 💯

**Review by mipro98 — APPROVED** [review]
🚀🚀🚀

**Review by maybeanerd — APPROVED** [review]
LGTM!

**Pull request merged**